internal/api/tests/private/highLevelConnectionStatus_test.go

```diff
@@ -41,8 +41,11 @@ var _ = Describe("high level connection status", func() {
 			OrgId: "12345",
 		}
 
-		result, response := getConnectionStatus(payload)
+		err, response := getConnectionStatus(payload)
+		if err != nil{
```

Comment:
The linter is complaining that the if statement is empty.

Can you try replacing the empty if statement with a check of the error like the following?

```
Expect(err).ToNot(HaveOccurred())
```